sdk/storage/storage-queue/package.json

```diff
@@ -49,8 +48,6 @@
   },
   "files": [
     "BreakingChanges.md",
-    "ThirdPartyNotices.txt",
```

Comment:
Why is this deleted? Published npm package doesn't require `ThirdPartyNotices.txt`?

Reply:
Without the bundled JS, there is no redistribution happening, so no TPN required 😁